src/main/resources/static/js/ChooseSource.js

```diff
@@ -148,104 +185,64 @@ function uploadUrl() {
 
 
 function preview() {
-    $('#previewbutton').prop('disabled', true);
+    $("#previewbutton").prop("disabled", true);
 
     $.ajax({
-        type: 'GET',
-        url: 'preview',
+        type: "GET",
+        url: "preview",
         data: {
             filename: currentFileName,
             headerLines: currentHeaderLines,
             delimiter: currentDelimiter
         },
         success: function (result) {
             loadPreview(result);
-            $('#previewbutton').prop('disabled', false);
+            $("#previewbutton").prop("disabled", false);
         },
         error: function (e) {
             addToLog(e.responseText);
-            $('#previewbutton').prop('disabled', false);
+            $("#previewbutton").prop("disabled", false);
         }
     });
 }
 
 function loadPreview(values) {
-    var tablebody = $('#previewTable').find('tbody:eq(0)');
-    var tablehead = $('#previewTable').find('thead:eq(0)');
-    tablehead.empty().append($('<tr>'));
+    var tablebody = $("#previewTable").find("tbody:eq(0)");
+    var tablehead = $("#previewTable").find("thead:eq(0)");
+    tablehead.empty().append($("<tr>"));
     tablebody.empty();
     var rows = values.length;
     if (rows > 0) {
         var columns = values[0].length;
     }
     for (var i = 0; i < rows; i++) {
-        tablebody.append($('<tr>'));
-        var current = tablebody.find('tr').last();
+        tablebody.append($("<tr>"));
+        var current = tablebody.find("tr").last();
         for (var j = 0; j < columns; j++) {
-            current.append($('<td>')
+            current.append($("<td>")
                 .text(values[i][j])
             );
         }
     }
 
-    current = tablehead.find('tr').last();
+    var current = tablehead.find("tr").last();
     for (var k = 0; k < columns; k++) {
-        current.append($('<th>')
-            .text('Column ' + k)
+        current.append($("<th>")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['current' used outside of binding context. (block-scoped-var)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)